src/reflect/scala/reflect/internal/Mirrors.scala

```diff
@@ -197,29 +197,35 @@ trait Mirrors extends api.Mirrors {
         case _                                                   => MissingRequirementError.notFound("package " + fullname)
       }
 
-    @deprecated("Use overload that accepts a String.", "2.13.0")
+    @deprecated("use overload that accepts a String.", since = "2.13.0")
     def getPackage(fullname: TermName): ModuleSymbol =
       getPackage(fullname.toString)
     def getPackage(fullname: String): ModuleSymbol =
       ensurePackageSymbol(fullname, getModuleOrClass(fullname, newTermName(_)), allowModules = true)
 
+    @deprecated("use overload that accepts a String.", since = "2.12.11")
     def getPackageIfDefined(fullname: TermName): Symbol =
+      getPackageIfDefined(fullname.toString)
+    def getPackageIfDefined(fullname: String): Symbol =
       wrapMissing(getPackage(fullname))
 
-    @deprecated("use getPackage", "2.11.0") def getRequiredPackage(fullname: String): ModuleSymbol =
-      getPackage(newTermNameCached(fullname))
+    @deprecated("use getPackage", "2.11.0")
+    def getRequiredPackage(fullname: String): ModuleSymbol =
+      getPackage(fullname)
 
-    def getPackageObject(fullname: String): ModuleSymbol = getPackageObject(newTermName(fullname))
+    @deprecated("use overload that accepts a String.", since = "2.12.11")
```

Comment:
The irony is that "anything goes" in `scala.reflect`!

Reply:
Name table's full, no dots allowed.